js/components/Toolbar.jsx

```diff
@@ -87,6 +64,7 @@ var Toolbar = React.createClass({
           <Link to="profile" params={{userId: this.state.currentUser._id}}>{this.state.currentUser.firstname}</Link>
```

Comment:
Ahh, the code editing hid it but it looks like we actually do still check for the currentUser.